tests/hopr.py

```diff
@@ -352,3 +375,40 @@ async def ticket_price(self):
         """
         _, response = self.__call_api(NetworkApi, "price")
         return int(response.price) if hasattr(response, "price") else None
+
+    async def startedz(self):
+        """
+        Checks if the node is started.
+        :return: started: int
+        """
+        url = f"{self.url}/startedz"
+        header = {"X-Auth-Token": self.token}
+        success = False
+
+        while success is False:
+            try:
+                response = requests.get(url, headers=header, timeout=1)
+                print(f"{response=}")
+                success = response.status_code == 200
+            except Exception:
+                success = False
+
+        return success
```

Comment:
If there's a bug inside the hoprd which will cause it to crash before being ready, it will hang tests indefinitely, this needs to be done for a specific period of time and then timeout.

Reply:
that's why in the conftest the call is wrapped in a "wait_for" which will cancel the the task an raise an error after 60s.